<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QWidget" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #b3dbd3</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="backMain">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white</string>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setNewButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white</string>
     </property>
     <property name="text">
      <string>Reset Details</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordInputBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameInputBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="usernameTitle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordTitle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoveBuddyTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;MoveBuddy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewDetailsButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white</string>
     </property>
     <property name="text">
      <string>View Details</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="privacyInputBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="privacyTitle">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Privacy</string>
     </property>
    </widget>
    <widget class="QLabel" name="locationTitle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="locationInputBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Error! Please contact admin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailInputBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="emailTitle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <zorder>background</zorder>
    <zorder>backMain</zorder>
    <zorder>setNewButton</zorder>
    <zorder>passwordInputBox</zorder>
    <zorder>usernameInputBox</zorder>
    <zorder>usernameTitle</zorder>
    <zorder>passwordTitle</zorder>
    <zorder>MoveBuddyTitle</zorder>
    <zorder>viewDetailsButton</zorder>
    <zorder>privacyInputBox</zorder>
    <zorder>privacyTitle</zorder>
    <zorder>locationTitle</zorder>
    <zorder>locationInputBox</zorder>
    <zorder>errorLabel</zorder>
    <zorder>emailInputBox</zorder>
    <zorder>emailTitle</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
